src/pages/our_ambassadors.astro

```diff
@@ -380,6 +380,20 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         of Nextflow in clinical and research settings.
       </AmbassadorCard>
 
+      <AmbassadorCard
+        name="Franz AKE"
+        img="Franz_ake.png"
+        country="fr"
+        github="Franzx7"
+        linkedin="franz-arnold-ake-3a657b11b"
+        twitter="franz_ake"
+      >
+        Franz AKE is a Bioinformatician specializing in single-cell transcriptomics. He completed his PhD in Bioinformatics at
+        <a href="https://p-cmrc.cat/research/plass-group/" target="_blank" 
+        > IDIBELL Institute </a> (Barcelona, SPAIN). He advocates for workflow automation and the integration of Nextflow
```

Comment:
It's still a tiny bit long. I tested here and if you remove `(Barcelona, SPAIN)`, the social media icons appear right, but no pressure. Feel free to shorten it the way you you prefer.

```suggestion
        > IDIBELL Institute</a>. He advocates for workflow automation and the integration of Nextflow
```